js/index.js

```diff
@@ -1,26 +1,40 @@
-const today = new Date();
-const thisYear = today.getFullYear();
-const footer = document.querySelector('footer');
-const copyRight = document.createElement('p');
+// copyright on footer
+const renderCopyright = () => {
+    const today = new Date();
+    const thisYear = today.getFullYear();
+
+    const copyright = document.querySelector('#copyright')
+    
+    copyright.innerHTML = `&copy; Raissa Gomes ${thisYear}`
+
+}
+
+renderCopyright();
+
+//skills section
+const renderSkillsList = () => {
+const skills = ['JavaScript', 'HTML', 'CSS', 'GIT']
 
-copyRight.innerHTML = `&copy Raissa Gomes ${thisYear}`;
-footer.appendChild(copyRight);
 
-const skills = ['HTML' 'CSS' 'JavaScript'];
+//DOM
 const skillsSection = document.getElementById('skills');
 const skillsList = skillsSection.querySelector('ul')
 
 for (let i = 0; i < skills.length; i++) {
     const skill = document.createElement('li');
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
+
+ }
```

Comment:
The skills are currently being added correctly, but it's easy to miss them since they're rendered as black text on black background:  

![](https://user-images.githubusercontent.com/9094098/210590782-11bb2081-b088-41fd-895e-91e831dbfafc.png)

If you want them to be styled the way your CSS is targeting, you can add them to the DOM with the `skl` class already set:

```suggestion
for (let i = 0; i < skills.length; i++) {
    const skill = document.createElement('li');
    skill.innerHTML = skills[i];
    skill.className = 'skl';
    skillsList.appendChild(skill);

 }
```